fields/mission-v1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Mission Statement",
+  "description": "The mission, purpose or objective of this entity",
+  "type": "string",
+  "maxLength": 1000,
+  "context": "http://purl.org/essglobal/vocab/mission"
```

Comment:
Note that only the `context` field is not described in the JSON Schema specification, so it will simply be ignored by JSON Schema parsers/validators.